tests/components/zha/test_device.py

```diff
@@ -185,7 +183,7 @@ async def test_check_available_unsuccessful(
 ):
     """Check device availability all tries fail."""
 
-    # pylint: disable=protected-access
+    # pylint: disable-next=protected-access
     zha_device = await zha_device_restored(device_with_basic_channel)
```

Comment:
Does this raise? You removed it above -> L134

Reply:
Obsolete, removed